ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -113,4 +114,51 @@ export const LivechatEnterprise = {
 
 		return LivechatTag.createOrUpdateTag(_id, tagData, tagDepartments);
 	},
+
+	savePriority(_id, priorityData) {
+		check(_id, Match.Maybe(String));
+
+		check(priorityData, {
+			name: String,
+			description: Match.Optional(String),
+			color: String,
+			dueTimeInMinutes: String,
+		});
+		LivechatRooms.updatePriorityDataByPriorityId(_id, priorityData);
```

Comment:
If I'm not wrong, this method will be processed even before a priority be created, am I right?
I think the `LivechatRooms`, `Subscriptions` and `LivechatInquiry` need to be processed after the priority creating/updating process succeed and then, just update the models mentioned before if you're updating a priority, otherwise, the priority just didn't exist before and due to that you don't update those models.